app/views/posts/show.html.erb

```diff
@@ -1,3 +1,8 @@
 <h1><%= @post.title %></h1>
 <p>By: <%= @post.author.name %></p>
-<p><%= @post.text %></p>
+<p><%= @post.content %></p>
+<p>Comments: <%= @post.comments.count %>, Likes: <%= @post.likes.count %></p>

```

Comment:
- [ ] I was unable to access all the posts for a particular user, perhaps because you mistakenly reference `@post.content` instead of `@post.text`.